robot/tests/test_network.robot

```diff
@@ -0,0 +1,141 @@
+*** Settings ***
+Documentation		This suite will verify network and MAC configurations
+...                     of OpenPower using ipmitool
+
+Resource                ../lib/ipmi_client.robot
+Resource                ../lib/utils.robot
+Resource                ../lib/connection_client.robot
+
+Suite Setup             Open Connection And Log In
+Suite Teardown          Close All Connections
+
+Test Setup              Pre Test Execution Setup
+Test Teardown           Expected Result verification
+
+*** Variables ***
+${IP_ADDREDD_INVALID}      9.3.181
+${IP_ADDREDD_STRING}       aa.bb.cc.dd
+${MAC_ADDREDD_VALID}       00:1a:3b:4e:5c:0a
```

Comment:
What happens when I have 100 openpower machines on my network and they're all running this test simultaneously?